roles/researchdata/molecule/default/verify.yml

```diff
@@ -0,0 +1,31 @@
+---
+# This is an example playbook to execute Ansible tests.
+
+- name: Verify
+  hosts: all
+  gather_facts: false
+  tasks:
+  - name: test for presence sudoers lines
+    lineinfile:
+      path: "/etc/sudoers"
+      line: "{{ item }}"
+      state: present
+    loop:
+      - "deploy ALL=(ALL) NOPASSWD: /usr/sbin/service nginx *"
+      - "Runas_Alias WWW = www-data"
+      - "deploy ALL = (WWW) NOPASSWD: ALL"
+      - "deploy ALL=(ALL) NOPASSWD: /bin/chown -R www-data /var/www/researchdata*"
+      - "deploy ALL=(ALL) NOPASSWD: /bin/chown -R deploy /var/www/researchdata*"
+    check_mode: true
+    register: present
+    failed_when:
+      - present is changed
+  - name: test for postgres database and login
+    command: psql -U researchdata -w researchdata -c '\d'
+    register: postgres_results
+    ignore_errors: true
+    changed_when: false
+  - name: assert cron jobs exists
```

Comment:
better label maybe? we are asserting a DB exists.

Reply:
Updated...